install/install.sh

```diff
@@ -337,12 +337,15 @@ print_install() {
       nushell )
         # shellcheck disable=SC2088
         config_file="${BOLD}your nu config file${NO_COLOR} (find it by running ${BOLD}\$nu.config-path${NO_COLOR} in Nushell)"
-        config_cmd="mkdir ~/.cache/starship
-        starship init nu | save -f ~/.cache/starship/init.nu
-        source ~/.cache/starship/init.nu"
+        config_cmd="\`use ~/.cache/starship/init.nu\`"
```

Comment:
```suggestion
        config_cmd="use ~/.cache/starship/init.nu"
```

Reply:
should be good in 8d43f2ac2254bd054ddac4d806799703bcfa6512 :yum: